tests/collection_change_indices.cpp

```diff
@@ -363,6 +363,7 @@ TEST_CASE("collection_change: calculate() sorted") {
 
     auto all_modified = [](size_t) { return true; };
     auto none_modified = [](size_t) { return false; };
+    size_t npos = -1;
```

Comment:
All of the changes in this file are related to disambiguating the size_t/int64_t overloads of calculate() with initializer list arguments.